drivers/sensor/bmi270/bmi270.c

```diff
@@ -0,0 +1,676 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bosch_bmi270
+
+#include <drivers/i2c.h>
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+
+#include "bmi270.h"
+
+const uint8_t bmi270_config_file[] = {
+#include "bmi270_config_file.txt"
+};
+
+LOG_MODULE_REGISTER(bmi270, CONFIG_SENSOR_LOG_LEVEL);
+
+#define MSB_TO_DATA(lsb, msb) (int16_t)(((uint16_t) msb << 8) + (uint16_t)lsb)
+
+static int reg_read(uint8_t reg, uint8_t *data, uint16_t length,
+		    struct bmi270_dev *dev)
```

Comment:
Usually a variable named `dev` is a `struct device *`. To avoid confusion, I recommend renaming the type to `struct bmi270_data` and the argument to something similar

Reply:
Done.